core/shared/src/main/scala/org/http4s/package.scala

```diff
@@ -84,40 +81,4 @@ package object http4s {
   /** A stream of server-sent events */
   type EventStream[F[_]] = Stream[F, ServerSentEvent]
 
-  // Syntax to enable JS-cross compilation without splitting files
-  // Nobody will actually be able to run these methods b/c they will fail to link
```

Comment:
I hope you don't mind, I also took this opportunity to remove these shims and any deprecated methods that needed them.